core/authorship/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "substrate-authorship"
```

Comment:
let's call it `substrate-basic-authorship` or similar, otherwise it gives the impression that it's the only/preferred way of authorship.